lms/djangoapps/verify_student/models.py

```diff
@@ -258,6 +258,28 @@ def verification_valid_or_pending(cls, user, earliest_allowed_date=None, queryse
         ).order_by('-created_at')
 
     @classmethod
+    def get_expiration_datetime(cls, user, queryset=None):
+        """
+        Check whether the user has an approved verification and Return the
+        "expiration_datetime" of most recent "approved" verification.
+
+        Arguments:
+            user (Object): User
+            queryset: If a queryset is provided, that will be used instead
+                of hitting the database.
+
+        Returns:
+            expiration_datetime: expiration_datetime of of most recent "approved"
+            verification.
+        """
+        if queryset is None:
+            queryset = cls.objects.filter(user=user)
+
+        queryset = queryset.filter(status='approved').order_by('-updated_at')
```

Comment:
You can use `.first()` method to get the first one of the result. https://docs.djangoproject.com/en/1.10/ref/models/querysets/#first